srcs/ServerCmds/privmsg.cpp

```diff
@@ -31,7 +31,7 @@ void Server::sendPrivmsgToUser(const Client *const client,
 	const std::string &targetName, const std::string &privMessage) {
 	const Client *const receiver = _clientMap.getClient(targetName);
 	if (receiver == NULL)
-		Utils::sendFormattedMessage(ERR_NOSUCHNICK, client);
+		Utils::sendFormattedMessage(ERR_NOSUCHNICK, client, targetName);
```

Comment:
change <channelName> by <arg> in define, remove const beforre client and set the last argument of the client